molecule/multi-ksql-connect-rhel/molecule.yml

```diff
@@ -176,6 +176,8 @@ provisioner:
               tasks.max: "1"
               file: "/etc/kafka/connect-distributed.properties"
               topics: "test_topic"
+              key.converter: "org.apache.kafka.connect.json.JsonConverter"
```

Comment:
Personal doubt:
What do these properties help with? Any utility for us?

Reply:
yup, as there's no schema registry configured in this test, we need to explicitly mention key and value converter classes